CHANGELOG.md

```diff
@@ -3,6 +3,36 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# [1.2.0](https://github.com/hyperledger/cacti/compare/v1.1.3...v1.2.0) (2023-03-28)
+
+### Bug Fixes
+
+* broken links ([1e8d68e](https://github.com/hyperledger/cacti/commit/1e8d68e7c92ec974d7cab5b124b9410949a86712))
```

Comment:
Is this auto-generated? seems it is capturing quite old commits from weaver. Is this intentional?
Or we can just say that `weaver` codebase merge into `cacti`?

Reply:
Actually, these links seem to point to the documentation fixes we made in response to the HackerOne bug report.